config/routes.rb

```diff
@@ -127,6 +127,16 @@
         post '/complete' => 'contact_details/review#complete', as: :contact_details_complete
       end
 
+      scope '/gcse' do
+        get '/maths/grade' => 'gcse/maths/grade#edit', as: :gcse_maths_edit_grade
+        get '/science/grade' => 'gcse/science/grade#edit', as: :gcse_science_edit_grade
+        get '/english/grade' => 'gcse/english/grade#edit', as: :gcse_english_edit_grade
+
+        patch '/maths/grade' => 'gcse/maths/grade#update', as: :gcse_maths_update_grade
```

Comment:
No need for the additional path helpers here, we can just PATCH the edit paths above and they'll resolve the same.

Reply:
Sorry @malcolmbaig - not sure I understand. Are you saying the path helpers (e.g. `:gcse_maths_update_grade`) should be deleted?

Reply:
Yes - the `gcse_maths_edit_grade_path` helper (for example) already gives you access to the '/maths/grade' endpoint, so the HTML form you build can just do a `method: :patch` and use the edit helper. This lines up with the path conventions Rails uses when it generates routes for you via the `resources` method, IIRC.

Reply:
👌